### ISSUE TITLE
scratch extension suggestion

### ISSUE THREAD

**Issue by 28elis** [Mrcomputer1Extensions/Mrcomputer1Extensions#2]
I was thinking of an extension that could take a screenshot of the current stage and save it as a costume in the sprite that the block was used in. Would this be possible?

**Comment by Mrcomputer1** [comment]
I don't think it would be possible as an extension (at least not without a mod).
